cfme/tests/control/test_basic.py

```diff
@@ -404,6 +426,16 @@ def test_container_image_compliance_policy_crud(soft_assert):
 
 
 @pytest.mark.tier(3)
+@pytest.mark.uncollectif(
+    lambda random_policy: random_policy in (explorer.ReplicatorCompliancePolicy,
+                                            explorer.PodCompliancePolicy,
+                                            explorer.ContainerNodeCompliancePolicy,
+                                            explorer.ContainerImageCompliancePolicy,
+                                            explorer.ReplicatorControlPolicy,
+                                            explorer.PodControlPolicy,
+                                            explorer.ContainerNodeControlPolicy,
+                                            explorer.ContainerImageControlPolicy) and
```

Comment:
Pull this list out from this lambda so it lies in this module and make it a set. Name it eg. `CONTAINER_POLICIES` so it is obvious :)